bin.js

```diff
@@ -4,12 +4,12 @@ const boxen = require('boxen')
 
 const argv = process.argv.slice(2)
 if (argv.length === 0) {
-  console.log('Please specify the wanted package manager: only-allow <npm|pnpm|yarn>')
+  console.log('Please specify the wanted package manager: only-allow <npm|cnpm|pnpm|yarn>')
   process.exit(1)
 }
 const wantedPM = argv[0]
-if (wantedPM !== 'npm' && wantedPM !== 'pnpm' && wantedPM !== 'yarn') {
-  console.log(`"${wantedPM}" is not a valid package manager. Available package managers are: npm, pnpm, or yarn.`)
+if (wantedPM !== 'npm' && wantedPM !== 'cnpm' && wantedPM !== 'pnpm' && wantedPM !== 'yarn') {
+  console.log(`"${wantedPM}" is not a valid package manager. Available package managers are: npm, cnpm, pnpm, or yarn.`)
   process.exit(1)
 }
 const usedPM = whichPMRuns()
```

Comment:
I am not sure this library currently detects cnpm

Reply:
CNPM is the domestic mirror library of China.  Since access to foreign networks is not allowed in China, the CNPM used is exactly the same as NPM, which is the domestic mirror of NPM in China and is used by many people